org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -144,12 +147,13 @@
 		private final Keyword cLeftParenthesisKeyword_1 = (Keyword)cGroup.eContents().get(1);
 		private final Keyword cRightParenthesisKeyword_2 = (Keyword)cGroup.eContents().get(2);
 		
-		//fragment FunctionHeader <Yield, Generator> *:
+		//fragment FunctionHeader <Yield,Generator>*:
```

Comment:
@ArneDeutsch Missing formatting rule for the parameter declarations. There is a space missing after the `,`

Reply:
Should be fixed with https://github.com/eclipse/xtext-core/issues/1489